### PR TITLE
Feature: View admin home

### PR DESCRIPTION
**Pull request by jrdiazr**
New feature, View for admin users where they are to list and filter the properties published

### CODE REVIEW

File: liveHomeFrontend/src/components/FilterBar.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import Button from './Button';
+import SearchBar from './SearchBar';
+import useInputValue from './../hooks/useInputValue';
+import styles from '../styles/components/FilterBar.module.sass';
+
+const FilterBar = () => {
+  const search = useInputValue('');
+  return (
+    <div className={styles.__container}>
+      <Button value='Filter' buttonType='search' buttonClass='grayLinearButton' />
```
Comment (lcasallas) [inline liveHomeFrontend/src/components/FilterBar.js:11]: Replace button for Modal component

File: liveHomeFrontend/src/styles/pages/admin/home.module.sass [hunk 2]
```diff
@@ -0,0 +1,30 @@
+@import '../../vars'
+
+.__container 
+  width: 90%
+  margin: 20px auto
```
Comment (helbertoro) [inline liveHomeFrontend/src/styles/pages/admin/home.module.sass:5]: this property should use the $spacing variable

File: liveHomeFrontend/src/styles/pages/admin/home.module.sass [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+.__founds
+  margin-left: auto
+  text-align: right
+  width: fit-content
+  font-size: 1.5rem
+  font-weight: 400
+  background: $background-field
+  padding: 5px 10px
```
Comment (helbertoro) [inline liveHomeFrontend/src/styles/pages/admin/home.module.sass:14]: this property should use the $spacing variable

File: liveHomeFrontend/src/styles/pages/admin/home.module.sass [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+.__card__container
+  display: grid
+  grid-template-columns: 1fr 1fr
+  grid-gap: 10px
+
+.__card__header
+  display: flex
+  justify-content: space-between
+  padding-right: 10px
```
Comment (helbertoro) [inline liveHomeFrontend/src/styles/pages/admin/home.module.sass:25]: this property should use the $spacing variable

File: liveHomeFrontend/src/styles/pages/admin/home.module.sass [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  align-items: center
+
+.property__name, .publish__date
+  margin: 5px 10px
```
Comment (helbertoro) [inline liveHomeFrontend/src/styles/pages/admin/home.module.sass:29]: this property should use the $spacing variable

File: liveHomeFrontend/src/components/SearchBar.js [hunk 6]
```diff
@@ -5,7 +5,10 @@ import styles from '../styles/components/SearchBar.module.sass';
 const SearchBar = (props) => {
   return (
     <form className={styles.searchBar__container} onSubmit={props.handleSubmit}>
-      <input className={styles.searchBar__container_input} type='text' placeholder='Search' name={props.name} value={props.value} />
+      <input
+        className={styles.searchBar__container_input} type='text' placeholder='Search' name={props.name} value={props.value}
+        onChange={props.onChange}
+      />
       <button className={styles.searchBar__container_button} type='submit'>
         <img src='/static/searchIcon.png' alt='icon search button' />
```
Comment (helbertoro) [inline liveHomeFrontend/src/components/SearchBar.js:13]: you should import image in the top, and use the variable name

**Comment by helbertoro** [comment]
the pull has conflict with other component

**Pull request merged**